### ISSUE TITLE
Statistical count() seems erratic

### ISSUE THREAD

**Issue by naomap** [drolbr/Overpass-API#567]
I am using the overpass-api.de server. 
The following produces a count of 49 shops :
```
nwr[shop](47.209,-1.552,47.215,-1.544);
make stat
  num = count(nodes);
out;
```

Strangely using count(nwr) produces a count of 0: 
```
nwr[shop](47.209,-1.552,47.215,-1.544);
make stat
  num = count(nwr);
out;

```
Using a named result set also produces 0:
```
nwr[shop](47.209,-1.552,47.215,-1.544)->.res;
make stat
  num = .res.count(nodes);
out;
```

**Comment by mmd-osm** [comment]
Thanks for reporting. This seems to be an issue with an incomplete list in: https://github.com/drolbr/Overpass-API/blob/oversized_backend_objects/src/overpass_api/statements/aggregators.cc#L485

**Comment by mmd-osm** [comment]
You can test the patch here, by the way: http://overpass-turbo.eu/s/STH

**Comment by naomap** [comment]
The first issue (counting on nwr) is fixed indeed, however the second issue (using a named set) apparently isn't.

BTW More than examples the series is built as a series of tutorials. I plan to write some more next week, and when I'm done with it I'll put them on my wiki next to the uMap series :-) I'll reference them on the Overpass documentation, and as they are published under CC-by-sa they can be translated.

**Comment by mmd-osm** [comment]
Well, the second one isn't really a bug.

Your query has one "." (dot) too much right in front of "res", Note that I've used `res.count(nodes);` rather than **.**`res.count(nodes);` in my example. This is according to the [documentation](https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL#Statistical_Count). As you can see [here](https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL#Sets), the "." is not part of the name of a set.

**Comment by naomap** [comment]
Oops sorry ! Didn't pay enough attention. Thanks for the clarification !
I can close it then :-)

**Comment by mmd-osm** [comment]
Please don't close this issue yet, this still needs to be fixed in the mater branch.

**Comment by drolbr** [comment]
Thank you for reporting the bug. Fixed in eb200aebdae27bd8635c5829a2bdfcdb06d0ab5c
